src/test/java/net/syscon/elite/repository/jpa/repository/VisitInformationRepositoryTest.java

```diff
@@ -0,0 +1,57 @@
+        assertThat(visits).extracting(VisitInformation::getEventStatus).containsOnly("CANC", null, null, null, null, null, null, null, null, null, null, null, null, null, "CANC");
+        assertThat(visits).extracting(VisitInformation::getEventStatusDescription).containsOnly("Cancelled", null, null, null, null, null, null, null, null, null, null, null, null, null, "Cancelled");
+        assertThat(visits).extracting(VisitInformation::getEventOutcome).containsOnly("ABS", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ATT", "ABS");
+        assertThat(visits).extracting(VisitInformation::getEventOutcomeDescription).containsOnly("Absence",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Attended",  "Absence");
+        assertThat(visits).extracting(VisitInformation::getStartTime).containsOnly(LocalDateTime.parse("2017-12-10T14:30"), LocalDateTime.parse("2017-11-13T14:30"), LocalDateTime.parse("2017-10-13T14:30"), LocalDateTime.parse("2017-09-15T14:00"), LocalDateTime.parse("2017-09-12T14:30"), LocalDateTime.parse("2017-09-10T14:30"), LocalDateTime.parse("2017-08-10T14:30"), LocalDateTime.parse("2017-07-10T14:30"), LocalDateTime.parse("2017-06-10T14:30"), LocalDateTime.parse("2017-05-10T14:30"), LocalDateTime.parse("2017-04-10T14:30"), LocalDateTime.parse("2017-03-10T14:30"), LocalDateTime.parse("2017-02-10T14:30"), LocalDateTime.parse("2017-01-10T14:30"), LocalDateTime.parse("2016-12-11T14:30"));
+        assertThat(visits).extracting(VisitInformation::getEndTime).containsOnly(LocalDateTime.parse("2017-12-10T15:30"),LocalDateTime.parse("2017-11-13T15:30"),LocalDateTime.parse("2017-10-13T15:30"),LocalDateTime.parse("2017-09-15T16:00"),LocalDateTime.parse("2017-09-12T15:30"),LocalDateTime.parse("2017-09-10T15:30"),LocalDateTime.parse("2017-08-10T15:30"),LocalDateTime.parse("2017-07-10T15:30"),LocalDateTime.parse("2017-06-10T15:30"),LocalDateTime.parse("2017-05-10T16:30"),LocalDateTime.parse("2017-04-10T15:30"),LocalDateTime.parse("2017-03-10T16:30"),LocalDateTime.parse("2017-02-10T15:30"),LocalDateTime.parse("2017-01-10T15:30"),LocalDateTime.parse("2016-12-11T15:30"));
+        assertThat(visits).extracting(VisitInformation::getLocation).containsOnly("Visiting Room", "Visiting Room", "Visiting Room", "Chapel", "Visiting Room", "Visiting Room", "Visiting Room", "Visiting Room", "Visiting Room", "Chapel", "Visiting Room", "Chapel", "Visiting Room", "Visiting Room", "Visiting Room");
+        assertThat(visits).extracting(VisitInformation::getVisitType).containsOnly("SCON", "SCON", "SCON", "OFFI", "SCON", "SCON", "SCON", "SCON", "SCON", "OFFI", "SCON", "OFFI", "SCON", "SCON", "SCON");
```

Comment:
`containsOnly` ignores duplicates so this would still pass if you removed all but one of the `"SCON"` values. If you want to ensure that all duplicates are present, you can use `containsExactly` if the order matters or `containsExactlyInAnyOrder` if it doesn't

Reply:
That's a very good suggestion, updated :)